metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/search_test.clj

```diff
@@ -423,32 +423,33 @@
             :errors ["Finding concept type [tag-association] with parameters [provider-id] is not supported."]}
            (util/find-concepts :tag-association {:provider-id "REG_PROV"})))))
 
-(deftest find-services
-  (let [serv1 (util/create-and-save-service "REG_PROV" 1 3)
-        serv2 (util/create-and-save-service "SMAL_PROV1" 2 2)]
-    (testing "find latest revsions"
-      (are2 [servs params]
-            (= (set servs)
-               (set (->> (util/find-latest-concepts :service params)
-                         :concepts
-                         (map #(dissoc % :revision-date :transaction-id)))))
-            "with metadata"
-            [serv1 serv2] {}
-
-            "exclude metadata"
-            [(dissoc serv1 :metadata) (dissoc serv2 :metadata)] {:exclude-metadata true}))
-
-    (testing "find all revisions"
-      (let [num-of-servs (-> (util/find-concepts :service {})
-                            :concepts
-                            count)]
-        (is (= 5 num-of-servs))))))
-
-(deftest find-services-with-invalid-parameters
-  (testing "extra parameters"
-    (is (= {:status 400
-            :errors ["Finding concept type [service] with parameters [short-name] is not supported."]}
-           (util/find-concepts :service {:short-name "SN1"})))))
+;; CMR-4173 should update
```

Comment:
Hehe. Good idea; I should have done the same thing for the service removal branch ;-)